source/features/conversation-activity-filter.tsx

```diff
@@ -181,6 +198,10 @@ void features.add(import.meta.url, {
 	additionalListeners: [
 		onConversationHeaderUpdate,
 	],
+	shortcuts: {
+		h: 'Hide/show events and collapsed comments',
```

Comment:
Is the naming clear?

Reply:
How about "Toggle hiding events and collapsed comments"? Same for the one below.

Reply:
New (just one key option) `Hide events or/and collapsed comments` any better idea?

Reply:
How about simply "Cycle between conversation activity filters"?